src/app/components/BibPage/BibPage.jsx

```diff
@@ -107,17 +107,21 @@ class ItemPage extends React.Component {
       { label: 'Number available', field: 'numAvailable' },
       { label: 'Number of items', field: 'numItems' },
       { label: 'Shelf Mark', field: 'shelfMark' },
-    ]
+    ];
+    let shortenItems = true;
+    if (this.props.location.pathname.indexOf('all') === -1) {
```

Comment:
so when the url has `all`, it has 20 items, but when it does not has `all`, it has all such as 2000 items?